tests/kernel/grub_baremetal.pm

```diff
@@ -0,0 +1,35 @@
+# SUSE's openQA tests
+#
+# Copyright © 2020 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Handle grub menu on ipmi console
+# Maintainer: Michael Moese <mmoese@suse.com>
+
+use strict;
+use warnings;
+use base "opensusebasetest";
+use testapi;
+use utils;
+
+sub run {
+    my ($self) = @_;
+    return unless check_var('BACKEND', 'ipmi');
```

Comment:
This shouldn't be needed, decision about module inclusion should be one step ahead. In our case it is yaml schedule which is only for bare metal.